Remotion/Web/Development.WebTesting/HostingStrategies/Configuration/TestSiteLayoutConfiguration.cs

```diff
@@ -41,6 +43,8 @@ public TestSiteLayoutConfiguration ([NotNull] IWebTestSettings webTestSettings)
       Resources = webTestSettings.TestSiteLayout.Resources
           .Select(resourceElement => EnsureRootedPath(RootPath, resourceElement))
           .Select(rootedPath => new TestSiteResource(rootedPath)).ToArray();
+      var pathToExecutable = Path.Combine(RootPath, webTestSettings.TestSiteLayout.PathFromRootToExecutable);
```

Comment:
It is already called Executable Path afterwards, and this is the `TestSiteLayoutConfiguration`, and this is the only usage where we read the config for any of these infos. I don't see the issue here.

Reply:
It's about expectations / easy of reading: "PathFromRootToExecutable" is a mouthful and you need to mentally parse the semantic of the config parameter's name and then use this information to build the correct value. This means everyone who sees this for the first time has to think about it. 

On the other hand, if we name it `ExecutablePath` here, too and require that the value is provided in its complete form, you see all the information in one place, no extra mental stack space required. 

I understand that the value is shorter with the current solution, but I'm not convinved that it's easier to maintain. 

Microsoft decided to do `processPath` in it's web.config
https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/web-config?view=aspnetcore-8.0

> Path to the executable that launches a process listening for HTTP requests. Relative paths are supported. If the path begins with `.`, the path is considered to be relative to the site root.

They do support both absolute and relative paths, so we could argue this should be an option. And since I only just stumbled (re-found) this setting: considering there's precedent, should we use `processPath` instead of ExecutablePath and mimick the semantics? (Note: the `.` semantic implies `.\bin\foo.exe` which is equivalent with `bin\foo.exe`, they just specified explicitly it for unknown purproses.)

And please remember: right now it is already possible to overrule the relative semantics by supplying an absolute path, so the naming is misleading anyway.

Reply:
I can work with processPath